backend/spec/helpers/admin/base_helper_spec.rb

```diff
@@ -14,4 +14,30 @@
       expect(datepicker_field_value(date)).to eq("2013/08/14")
     end
   end
+
+  describe "#admin_layout" do
+    subject { admin_layout(value) }
+
+    context "when an initial value is set" do
+      let(:value) { "full-width" }
+
+      it "sets and returns the value passed to it" do
+        expect(subject).to eq value
+      end
+
+      context "and you call it again without an argument" do
+        before { admin_layout(value) }
+        let(:value) { nil }
+
+        it "returns the initial value that was set" do
+          expect(subject).to eq value
+        end
+      end
+    end
```

Comment:
You're right that I shouldn't be testing that I get `value` back and instead should be making sure I get the exact value that I am expecting.